src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -59,7 +59,7 @@ class AppsResourceTest extends AkkaUnitTest with GroupCreation with JerseyTest {
       config.defaultNetworkName.toOption,
       config.mesosBridgeName(),
       config.availableFeatures,
-      ResourceRole.Unreserved,
+      config.mesosRole(),
```

Comment:
This fixes the test case but I'm not sure this is the issue here.

Reply:
It certainly is correct to use the config.mesosRole() here...

Reply:
That's not the point. The following app will not currently be accepted on the master started with `--mesos_role slave_public` (though it should):
```
{
   "id": "/sleep",
   "cmd": "sleep 3600"
   "acceptedResourceRoles": ["slave_public"]
}

Reply:
Technically, I think one could say the group-role for root is "*"; we may wish to consider allowing both "*" and the default Mesos role.